metering/reports/metering-storage-locations.adoc

```diff
@@ -1,17 +1,17 @@
 [id="metering-storage-locations"]
-= Storage Locations
+= Storage locations
 include::modules/common-attributes.adoc[]
 :context: metering-storage-locations
 
 toc::[]
 
-A StorageLocation is a custom resource that configures where data will be stored by the reporting-operator. This includes the data collected from Prometheus, and the results produced by generating a Report custom resource.
+A `StorageLocation` custom resource configures where data will be stored by the Reporting Operator. This includes the data collected from Prometheus, and the results produced by generating a `Report` custom resource.
 
-You only need to configure a StorageLocation if you want to store data in multiple locations, like multiple S3 buckets or both S3 and HDFS, or if you wish to access a database in Hive/Presto that was not created by metering. For most users this is not a requirement, and the xref:../../metering/configuring_metering/metering-about-configuring.adoc#metering-about-configuring[documentation on configuring metering] is sufficent to configure all necessary storage components.
+You only need to configure a `StorageLocation` custom resource if you want to store data in multiple locations, like multiple S3 buckets or both S3 and HDFS, or if you wish to access a database in Hive and Presto that was not created by metering. For most users this is not a requirement, and the xref:../../metering/configuring_metering/metering-about-configuring.adoc#metering-about-configuring[documentation on configuring metering] is sufficient to configure all necessary storage components.
```

Comment:
I think we should remove the mention of HDFS entirely here as that's not a supported storage backend, and I was under the impression we had removed all mentions of this a couple of releases ago.

Reply:
(that can be done in a follow-up PR though)

Reply:
@timflannagan I don't recall needing to remove mentions to HDFS, but can do so in another PR. I'll follow-up with you outside of this PR about it 👍